CHANGELOG.md

```diff
@@ -94,10 +94,12 @@
 - Convert admonition syntax from Markdown to MyST. @sneridagh
 - Make links build both in Volto and Plone documentation. See https://github.com/plone/volto/pull/3094 @stevepiercy
 - Fix broken links. @stevepiercy
+- Updated Installation Guide. @ktsrivastava29
```

Comment:
Volto 15.0.0 was released yesterday. Under the new 15.0.1 heading, would you please:

- create a subheading `### Documentation`, and
- move this item under that subheading?